drivers/include/drivers/InterruptIn.h

```diff
@@ -102,30 +102,30 @@ class InterruptIn : private NonCopyable<InterruptIn> {
      *
```

Comment:
why not also 
```cpp
virtual operator int();
```

Reply:
It calls read :wink:

Reply:
![giphy](https://user-images.githubusercontent.com/12911360/160819043-2949b148-93df-4775-a05f-64373ba607c5.gif)